apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -79,6 +80,12 @@ <h1>View Users</h1>
     else options.push(option.value)
 })
 
+for (const td of document.querySelectorAll("tr>td:nth-child(5)")) {
```

Comment:
I think you could do this in the view instead of in the template (in `_set_user`); generally, data processing is better to handle in the view rather than in the template. From there, in the modal template, to display True/False, you could display the value of `{{r.active == 'Active'}}`.

Reply:
Yeah, this makes sense to me!

Reply:
I removed the script in template and changed _set_user as suggested to handle the data processing. 
<img width="874" alt="Screenshot 2023-03-26 at 4 53 46 PM" src="https://user-images.githubusercontent.com/12832518/227813721-da2116e6-5e98-4fa9-a395-2b0e734df2cd.png">
<img width="711" alt="Screenshot 2023-03-26 at 4 54 04 PM" src="https://user-images.githubusercontent.com/12832518/227813728-0911dc94-3ef3-4aa7-b7dc-302e7ada6cd3.png">